templates/AMI-Template-Instances.cfn.yml.j2

```diff
@@ -46,8 +46,10 @@ Resources:
             DeleteOnTermination: true
             VolumeSize: {{ disk_size }}
             VolumeType: "gp2"
-      IamInstanceProfile: !Ref {% if layer != 'Solr' %}Web{% endif %}InstanceRoleProfile
-      ImageId: "ami-0d71fe73adf7a9887"
+      IamInstanceProfile:
```

Comment:
Is it possible to set the disks to 'Encrypted': true
so that it makes encrypted AMI's.

Reply:
Done, will test in DEV before proceeding.